package.json

```diff
@@ -19,5 +19,11 @@
     "stylelint-scss": "^3.17.2",
     "webpack": "^5.6.0",
     "webpack-cli": "^4.2.0"
+  },
+  "dependencies": {
+    "css-loader": "^5.0.1",
+    "sass": "^1.29.0",
+    "sass-loader": "^10.1.0",
+    "style-loader": "^2.0.0"
```

Comment:
- [ ] Please update the scripts in your package.json. Let it resemble something like this:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```